docs/packages/pkg/corrade.rst

```diff
@@ -0,0 +1,42 @@
+.. spelling::
+
+    corrade
+
+.. index::
+  single: graphics ; corrade
+
+.. _pkg.corrade:
+
+corrade
+===
+
+-  `Official <https://github.com/mosra/corrade>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/corrade/CMakeLists.txt>`__
+-  Developed by `Vladimír Vondruš<https://github.com/mosra>`
+-  Added by `Pascal Thomet <https://github.com/pthom>`__ (`pr-1646 <https://github.com/ruslo/hunter/pull/1646>`__)
+
+usage
```

Comment:
Please inject code from real `CMakeLists.txt`, see templates:
* https://github.com/ruslo/hunter/blob/v0.23.64/examples/foo/CMakeLists.txt#L12-L18
* https://raw.githubusercontent.com/ruslo/hunter/v0.23.64/docs/packages/pkg/foo.rst

Reply:
Fixed by eb4a7fbe74a2baed2acd7a6ac0113725cfc02c0a
Thanks for the advice

Reply:
done